File: Beecrowd/1066 Pares, Ímpares, Positivos e Negativos/main.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
	int X, i, Pos = 0, Neg = 0, Imp = 0, Par = 0;
	
	for(i = 0; i < 5; i++){
		cin >> X;
		if(X < 0)
			Neg++;
		else if(X > 0)
			Pos++;
		if(X % 2 == 0)
			Par++;
		else
			Imp++;
	}
	
	cout << Par << " valor(es) par(es)" << endl;
	cout << Imp << " valor(es) impar(es)" << endl;
	cout << Pos << " valor(es) positivo(s)" << endl;
	cout << Neg << " valor(es) negativo(s)" << endl;
 
    return 0;
}

File: Beecrowd/1066 Pares, Ímpares, Positivos e Negativos/main.py
Pos = 0
Neg = 0
Imp = 0
Par = 0
	
for i in range(5):
  X = int(input())
  if X < 0:
    Neg += 1
  elif X > 0:
    Pos += 1
  if X % 2 == 0:
    Par += 1
  else:
    Imp += 1
	
print("{} valor(es) par(es)".format(Par))
print("{} valor(es) impar(es)".format(Imp))
print("{} valor(es) positivo(s)".format(Pos))
print("{} valor(es) negativo(s)".format(Neg))